### PR TITLE
Cleanups and fix broken links

### PR DESCRIPTION
**Pull request by drewmoseley**

### CODE REVIEW

File: 03.Devices/02.Yocto-project/02.Bootloader-support/02.U-Boot/01.Manual-U-Boot-integration/01.Providing-custom-u-boot-fw-utils/docs.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ will go through each line and explain why they are needed.
     binaries that are specific to this board, and not to the architecture in
     general. This stems from the highly board specific nature of U-Boot. See
     [the official Yocto Project
-    documentation](http://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#var-PACKAGE_ARCH)
+    documentation](http://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#var-PACKAGE_ARCH?target=_blank)
```
Comment (estenberg) [inline 03.Devices/02.Yocto-project/02.Bootloader-support/02.U-Boot/01.Manual-U-Boot-integration/01.Providing-custom-u-boot-fw-utils/docs.md]: I think the target blank part needs to come before any hash, i.e.

http://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html?target=_blank#var-PACKAGE_ARCH

Would do a quick check to verify

**Comment by drewmoseley** [comment]
@estenberg I tested and you were correct.  I've fixed the ordering of the "#" and "?" in the URLs.

**Comment by estenberg** [comment]
@drewmoseley looks good! Would run a dead link checker after merge just to verify (hard to do manually).

It looks like a conflict snuck in though, but otherwise all clear to merge as I can see!

**Comment by estenberg** [comment]
We are still having some Travis failures:

The command "./test_docs.py 07.Administration/02.Production-installation/docs.md" exited with 0.

Looks unrelated but probably someone should look at that...

**Pull request merged**